### PR TITLE
change the menu from toolbar to swipeleft in CreateGame

### PR DESCRIPTION
**Pull request by DonnA98**

### CODE REVIEW

File: app/src/main/res/menu/menu_appbar.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    <group android:id="@+id/menu_findGame_group">
+        <item
+            android:id="@+id/menu_actionSearch"
+            android:orderInCategory="200"
+            android:title="@string/searchFilterSort"
+            android:icon="@drawable/magnifyingglass"
+            app:showAsAction="ifRoom"/>
+    </group>
+</menu>
```
Comment (Tinawyn) [inline app/src/main/res/menu/menu_appbar.xml]: Add a line

File: app/src/main/res/menu/menu_navigation.xml [hunk 2]
```diff
@@ -2,37 +2,25 @@
...
     </group>
 
     <item
         android:id="@+id/menu_myAccount"
         android:icon="@drawable/ic_account"
         android:orderInCategory="200"
-        android:title="@string/titleMyAccountActivity"
-        app:showAsAction="ifRoom" />
+        android:title="@string/titleMyAccountActivity"/>
 </menu>
```
Comment (Tinawyn) [inline app/src/main/res/menu/menu_navigation.xml]: Add a Line please

File: app/src/main/res/menu/menu_navigation.xml [hunk 3]
```diff
@@ -2,37 +2,25 @@
...
-    </group>
-
     <group android:id="@+id/menu_gm_group">
         <item
             android:id="@+id/menu_createGame"
             android:icon="@drawable/ic_quill"
             android:orderInCategory="200"
-            android:title="@string/title_create_game_activity"
-            app:showAsAction="ifRoom" />
+            android:title="@string/title_create_game_activity"/>
```
Comment (rykeuse) [inline app/src/main/res/menu/menu_navigation.xml]: Format using camel case like the others

Comment (DonnA98) [reply app/src/main/res/menu/menu_navigation.xml]: I thought it would be better to use snake case in the xml files since it's the xml convention

**Pull request merged**